amqp/src/main/scala/akka/stream/alpakka/amqp/javadsl/AmqpSink.scala

```diff
@@ -6,44 +6,43 @@ package akka.stream.alpakka.amqp.javadsl
 
 import java.util.concurrent.CompletionStage
 
-import scala.compat.java8.FutureConverters._
 import akka.Done
 import akka.stream.alpakka.amqp._
 import akka.util.ByteString
 
+import scala.compat.java8.FutureConverters._
+
 object AmqpSink {
 
   /**
-   * Java API: Creates an [[AmqpSink]] that accepts [[akka.stream.alpakka.amqp.WriteMessage WriteMessage]] elements.
+   * Creates an [[AmqpSink]] that accepts [[WriteMessage]] elements.
```

Comment:
Scaladoc requires fully-qualified class names (even though IDE's are happy without).

Reply:
I changed it to backticks, as you recommended for `AmqpFlow` in one of the previous comments.